orangecontrib/text/preprocess.py

```diff
@@ -1,94 +1,161 @@
-from nltk.stem.porter import PorterStemmer
+from nltk.corpus import stopwords
 from nltk.stem import WordNetLemmatizer
-from sklearn.feature_extraction.text import CountVectorizer
+from nltk.tokenize import word_tokenize
+from nltk.stem.porter import PorterStemmer
+from nltk.stem.snowball import SnowballStemmer
+
+from orangecontrib.text.corpus import Corpus
+
+import Orange
 
 
 class Preprocessor:
     """
-        Holds pre-processing flags and other information, about stop word
-        removal, lowercasing, text morphing etc.(the options are set via
-        the Preprocess widget).
+        A pre-processing class for the Orange text mining extension. An instance of the class
```

Comment:
`class for the Orange text mining extension. An instance of the class` can be deleted.